docs/docs/user-guides/restaking-sol.md

```diff
@@ -0,0 +1,17 @@
+# How to Restake your tokens 
+
+The [MANTIS app](https://games.mantis.app/) enables you to restake your assets effortlessly using intents. You will need a Phantom wallet to interact with the application. Just prompt the token and amount you want to stake or follow the suggestions. 
+
+After connecting your wallet, you will see the following screen. Currently, you can either restake bSOL, mSOL, jitoSOL or SOL (staking SOL occurs via converting SOL into wSOL and then transferring it to the vault). 
+
+![stake-dropdown](../user-guides/images-restaking-guide/restaking-1.png)
+
+Once your intent is expressed as displayed below, you’ll need to approve the transaction with your wallet. 
+
+:::info
+The fee to mint an NFT is 0.03 SOL, this amount will be returned to your account once the NFT is burned. You can read more about this process [here](../technology/solana-restaking/vaults.md).
```

Comment:
shall we explain what these NFTs are for?

Reply:
given that there are two NFTs, it can be confusing

Reply:
it's explained in the linked section, not in the user guide